cura/PrinterOutputDevice.py

```diff
@@ -117,6 +124,16 @@ def setConnectionState(self, connection_state: ConnectionState) -> None:
             self._connection_state = connection_state
             self.connectionStateChanged.emit(self._id)
 
+    def checkConnectionType(self, connection_type: ConnectionType) -> bool:
```

Comment:
uh? Why is this a function? This is essentially the same the getConnection...

Reply:
Removed in 0aa49270ebb53521b3364ce5ab795b51d3e3adba This function is not used.